common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -287,37 +288,20 @@ def __init__(
         """
         Create & open the connection, authenticate, and provide pointers to the collections
         """
-        if kwargs.get('replicaSet') is None:
-            kwargs.pop('replicaSet', None)
-            mongo_class = pymongo.MongoClient
-        else:
-            mongo_class = pymongo.MongoReplicaSetClient
-        _client = mongo_class(
-            host=host,
-            port=port,
-            tz_aware=tz_aware,
-            **kwargs
-        )
-        self.database = MongoProxy(
-            pymongo.database.Database(_client, db),
-            wait_time=retry_wait_time
-        )
+        # Set a write concern of 'majority', which makes writes complete successfully to a majority
+        # of nodes in the replica set before returning. Also makes pymongo report write errors.
+        kwargs['w'] = 'majority'
```

Comment:
I missed that this changed to majority.  Right now we do 1, so this would be a write performance regression I think (this is what we discussed changing to be configurable in a later PR).  For now, I believe this should go back to being 1.